request_logger.py

```
@@ -0,0 +1,105 @@
+        try:
+            if int(response.headers.get('content-length', 0)) < 10000:
+                try:
+                    response_data['content'] = response.json()
+                except:
+                    # Not JSON, so add a snippet of text or indicate binary
+                    content_type = response.headers.get('content-type', '')
+                    if 'text' in content_type or 'json' in content_type:
+                        response_data['content'] = response.text[:500] + ('...' if len(response.text) > 500 else '')
+                    else:
+                        response_data['content'] = '[binary data]'
+        except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)